# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: toolkit/components/backgroundhangmonitor/tests/xpcshell.toml
# Commit: bf92aff87824
# Full Hash: bf92aff878241e4f9e2b6df2a74f636685f7a3d5
# Author: Nazım Can Altınova <canaltinova@gmail.com>
# Date: 2025-05-31 08:59:27
# Description:
#   Bug 1966705 - Disable BHR on 32 bit Android due to EHABI stackwalker crashes r=mstange
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251923
# ==============================================================================

diff -r 7d5de260f508 -r bf92aff87824 toolkit/components/backgroundhangmonitor/tests/xpcshell.toml
--- a/toolkit/components/backgroundhangmonitor/tests/xpcshell.toml	Fri May 30 18:39:02 2025 +0000
+++ b/toolkit/components/backgroundhangmonitor/tests/xpcshell.toml	Fri May 30 18:42:56 2025 +0000
@@ -6,6 +6,7 @@
 run-sequentially = "very high failure rate in parallel"
 skip-if = [
   "debug",
+  "os == 'android' && processor == 'armeabi-v7a'", # Bug 1969481
   "os == 'mac' && os_version == '10.15' && processor == 'x86_64' && opt", # Bug 1417723
   "os == 'mac' && os_version == '11.20' && arch == 'aarch64' && opt", # Bug 1417723
   "os == 'mac' && os_version == '14.70' && processor == 'x86_64' && opt", # Bug 1417723
